near-sdk/tests/test-contracts/lazy/src/lib.rs

```diff
@@ -49,6 +58,19 @@ impl LazyContract {
         }
     }
 
+    /// This should write twice on each iteration.
+    #[payable]
+    pub fn insert_take_flush(&mut self, iterations: u32) {
```

Comment:
I would name it `insert_take` to be consistent with `insert_delete`.

Reply:
will do